quirks/shared-credentials.json

```diff
@@ -244,6 +244,16 @@
             "postnl.be"
         ]
     },
+    {
+        "from": [
+            "protonmail.ch",
+            "protonmail.com"
+        ],
+        "to": [
+            "proton.me"
+        ],
+        "fromDomainsAreObsoleted": false
```

Comment:
I think this property should be set to true in this case.

[Contributing guidelines](https://github.com/apple/password-manager-resources/blob/main/CONTRIBUTING.md) say this:

> fromDomainsAreObsoleted is a boolean that indicates whether or not the from domains in an entry have been obsoleted by the to domains (meaning logins are no longer served on the from domains). In the case of a redirect (such as [discordapp.com](https://discordapp.com/) -> [discord.com](https://discord.com/)), this would be true.